packages/react-core/src/components/DataToolbar/DataToolbarItem.tsx

```diff
@@ -18,10 +18,9 @@ export enum DataToolbarItemVariant {
 export interface DataToolbarItemProps extends React.HTMLProps<HTMLDivElement> {
   /** Classes applied to root element of the data toolbar item */
   className?: string;
-  /** A type modifier which modifies spacing specifically depending on the type of item */
+  /** TODO: Support 'separator' element as a <Divider component="div" variant="horizontal" />. A type modifier which modifies spacing specifically depending on the type of item */
   variant?:
     | DataToolbarItemVariant
-    | 'separator'
```

Comment:
Maybe your breaking change notes should specifically call out DataToolbarItem instead of DataToolbar?

Reply:
We're planning on adding support back DataToolbarItem's variant="separator" once Divider implements variant="horizontal".